src/Microsoft.AspNetCore.Blazor.Server/Circuits/BlazorHub.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Blazor.Services;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Blazor.Server.Circuits
+{
+    internal class BlazorHub : Hub
+    {
+        private static readonly object CircuitKey = "circuit";
```

Comment:
It's a minor point, but would the value `new object()` be preferable over `"circuit"` to make it less prone to outside interference?

Reply:
yeah it might be slightly better

Reply:
The advantage of this is that it's readable in the debugger. I might just give it a slightly more gross looking name.